### PR TITLE
converting label to camelCase

### PR DESCRIPTION
**Pull request by gaagul**
Made the suggested changes in #13.
Removed unnecessary div tags and created a function to convert labels to camelCase.

### CODE REVIEW

**Comment by gaagul** [comment]
Screenshot 2022-09-28 00-44-20

File: app/javascript/src/components/Dashboard/utils.js [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+        return word.toLowerCase();
+      }
+
+      return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+    })
+    .join("");
+
+export const buildSelectOptions = options =>
+  options.map(option => {
+    const value = toCamelCase(option);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/utils.js:17]: Unnecessary variable. You can remove this variable and do an implicit return.

File: app/javascript/src/components/Dashboard/utils.js [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+  str
+    .split(" ")
+    .map((word, index) => {
+      if (index === 0) {
+        return word.toLowerCase();
+      }
+
+      return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+    })
+    .join("");
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/utils.js:13]: If we provide a string like `UX-Designer`, then we ought to get `uxDesigner` as the output. Refactor accordingly.

**Review by yedhink — COMMENTED** [review]
@gaagul

**Pull request merged**